tests/xharness/Microsoft.DotNet.XHarness.iOS.Shared/TestImporter/Templates/Managed/XamariniOSTemplate.cs

```diff
@@ -27,7 +27,7 @@ public class XamariniOSTemplate : ITemplatedProject {
 		internal static readonly string TestingFrameworksKey = "%TESTING FRAMEWORKS%";
 
 		// resource related static vars used to copy the embedded src to the hd
-		static string srcResourcePrefix = "Xharness.TestImporter.Templates.Managed.Resources.src.";
+		static string srcResourcePrefix = "Microsoft.DotNet.XHarness.iOS.Shared.TestImporter.Templates.Managed.Resources.src.";
```

Comment:
This could also probably be resolved through reflection

Reply:
Enhancement I want to add :)